### PR TITLE
feat: use webpack to build html template

### LINKED ISSUE #3544: use webpack to build html template

### PR DESCRIPTION
**Pull request by myan9**
Fixes #3544

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [ ] 💅 Enhancement
- [x] 🚀 New feature

#### Please confirm that your PR fulfills these requirements

- [x] Multiple commits are squashed into one commit.
- [x] The commit message follows [Conventional Commits](https://github.com/IBM/kui/blob/master/CONTRIBUTING.md#conventional-commits), which allows us to autogenerate release notes; e.g. `fix(plugins/plugin-k8s): fixed annoying bugs`
- [x] All npm dependencies are pinned.

### CODE REVIEW

**Comment by starpit** [comment]
this also removes configure.js, nice!

File: packages/webpack/webpack.config.js [hunk 1]
```diff
@@ -190,59 +191,61 @@ console.log('entry', entry)
...
+  {
+    filename: path.join(outputPath, 'index.html'),
+    template: path.join(stageDir, 'node_modules/@kui-shell/core/templates/index.ejs')
+  }
+)
 
-          // and this will inject it
-          const Builder = require(path.join(builderHome, 'lib/configure'))
-          await new Builder().build(!inBrowser || isWatching ? 'webpack-watch' : 'webpack', overrides)
+console.error('htmlBuildOptions!!!', htmlBuildOptions)
```
Comment (starpit) [inline packages/webpack/webpack.config.js]: oops, leftover debugging message

Comment (starpit) [reply packages/webpack/webpack.config.js]: also, can't we remove the remaining two files in packages/builder/lib/?

Comment (starpit) [reply packages/webpack/webpack.config.js]: webpack/build.sh still refers to index-webpack.html

Comment (myan9) [reply packages/webpack/webpack.config.js]: solved, thanks!

**Pull request merged**